src/components/KSlideout/KSlideout.vue

```diff
@@ -13,10 +13,65 @@
       <div
         v-if="isVisible"
         class="panel"
-        :class="{ isVisible: 'is-visible' }"
+        :class="{ 'is-visible': isVisible, 'border-styles': !overlayEnabled }"
       >
+        <!-- If there is title prop, display the header content -->
+        <div
+          v-if="title"
```

Comment:
this conditional also needs to check for slot content - `title` could be left empty and only the `titleContent` slot is used which we should still allow.